web/src/app/modules/sugarloaf/components/smart/content/content.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="content-component">
+<div clrFocusOnViewInit #scrollTarget class="content-component">
```

Comment:
`scrollTarget` was removed as part of slider cleanup in https://github.com/vmware-tanzu/octant/commit/bcf1148c0f23811866969e124464a0ba3b093fd7#diff-4399b3274650f85b70c641d7e49db84d

Reply:
Removed